Classes/common/Encryption/Keychain/CDTEncryptionKeychainManager.m

```diff
@@ -0,0 +1,193 @@
+//
+//  CDTEncryptionKeychainManager.m
+//
+//
+//  Created by Enrique de la Torre Fernandez on 09/04/2015.
+//
+//
+//  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+//  except in compliance with the License. You may obtain a copy of the License at
+//  http://www.apache.org/licenses/LICENSE-2.0
+//  Unless required by applicable law or agreed to in writing, software distributed under the
+//  License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+//  either express or implied. See the License for the specific language governing permissions
+//  and limitations under the License.
+//
+
+#import <CommonCrypto/CommonCryptor.h>
+
+#import "CDTEncryptionKeychainManager.h"
+
+#import "CDTEncryptionKeychainUtils.h"
+#import "CDTEncryptionKeychainConstants.h"
+
+#import "CDTLogging.h"
+
+@interface CDTEncryptionKeychainManager ()
+
+@property (strong, nonatomic, readonly) CDTEncryptionKeychainStorage *storage;
+
+@end
+
+@implementation CDTEncryptionKeychainManager
+
+#pragma mark - Init object
+- (instancetype)init { return [self initWithStorage:nil]; }
```

Comment:
As suggested by @rhyshort, I use `UNAVAILABLE_ATTRIBUTE ` to make 'init' unavailable.

Fixed in 10d1a60.